.gherkin-lintrc

```diff
@@ -0,0 +1,74 @@
+{
+  "no-files-without-scenarios" : "on",
+  "no-unnamed-features": "on",
+  "no-unnamed-scenarios": "on",
+  "no-dupe-scenario-names": ["on", "in-feature"],
+  "no-dupe-feature-names": "on",
+  "no-partially-commented-tag-lines": "on",
+  "indentation" : [
+    "on", {
+      "Feature": 0,
+      "Background": 2,
+      "Scenario": 2,
+      "Step": 2,
+      "Examples": 4,
+      "example": 6,
+      "given": 4,
+      "when": 4,
+      "then": 4,
+      "and": 4,
+      "but": 4,
+      "feature tag": 2,
+      "scenario tag": 2
+    }
+  ],
+  "no-trailing-spaces": "on",
+  "new-line-at-eof": ["on", "no"],
```

Comment:
Enforcing no new line at EOF causes issues when feature files are edited in the github web editor. See [here](https://github.com/camaraproject/DeviceDataVolume/pull/40).

My preference would be for this rule to either be explicitly `off`, or just not included in the rules at all.

Reply:
OK, thanks for the explanation. Maybe they should have called the switch `new-line-character-at-eof`.

Nonetheless, a fix is needed to allow editing of feature files using the GitHub web editor. I have no strong opinion on whether this is enforcing the POSIX standard, or simply allowing each editor to do its own thing (though I do prefer standards 😃).